src/browserWindow.js

```diff
@@ -9,12 +9,14 @@ const { BrowserWindow } = require('electron');
 
 let settings = {};
 let transparency = false;
+let frame = true;
 let ewp = false;
 try {
-  settings = require(join(__dirname, '../../settings/pc-general.json'));
+  settings = require(join(__dirname, '../settings/pc-general.json'));
   transparency = settings.transparentWindow;
+  frame = settings.windowFrame;
   ewp = settings.experimentalWebPlatform;
-} catch (e) {}
+} catch (e) { console.error(e) }
```

Comment:
Failure here is to be expected if the user doesn't have a config file (e.g. on a clean install) which is why we ignore it silently.